Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```
@@ -293,11 +293,12 @@
             Slash: 12
             Heat: 12
             Structural: 15
+  # Disabled until the wield active sfx no longer stacks
   - type: ItemToggleActiveSound
     activeSound:
       path: /Audio/Weapons/ebladehum.ogg
       params:
```

```suggestion
  #- type: ItemToggleActiveSound
  #  activeSound:
  #    path: /Audio/Weapons/ebladehum.ogg
  #    params:
```
I would rather comment it out than reducing the volume